ChangeLog

```diff
@@ -85,6 +85,7 @@
  * #995 (Pip does not recognize conda install of openturns)
  * #1000 (SaltelliSensitivityAlgorithm with LowDiscrepancyExperiment produces wrong results)
  * #1005 (The stopping criteria of SobolSimulationAlgorithm is weird)
+ * #1006 (Text drawable does not handle size)
```

Comment:
It's a feature so it will be in the next release.

Reply:
fine with me. I update the ChangeLog

Reply:
+1 good job, small enough to be in 1.13? Really in next release?

Reply:
yup, can you update the changelog Sofiane ?

Reply:
Hmm didn't understand. Should I move it to another section?

Reply:
Yes to the 1.14 section

Reply:
thus it will not be integrated to 1.13 as @regislebrun mentionned?

Reply:
As it introduces new apis I prefer not to include it, sorry.

Reply:
ok, fine with me

Reply:
@jschueller done